common/djangoapps/third_party_auth/samlproviderconfig/views.py

```diff
@@ -82,3 +83,29 @@ def get_permission_object(self):
         can access these endpoints, we have to sort out the operator role use case
         """
         return self.requested_enterprise_uuid
+
+    def create(self, request, *args, **kwargs):
```

Comment:
Is this overriding the default create for the actual Provider Config? And if so is it still creating the actual provider config, because the below looks like it's just creating the association (via EnterpriseCustomerIdentityProvider)

Reply:
yup it creates the actual config due to the serializer call ` self.perform_create(serializer)` then I go on and pick up that newly created model and create the association from it ...  assuming I did this right and assuming the test is correct too ...